docs/modules/system preparation.rst

```diff
@@ -0,0 +1,309 @@
+.. _system_preparation:
+
+Preparing systems for constant-pH simulation
```

Comment:
Either way, this is useful for the moment as it also helps me set things up reproducibly. I can remove it later, once the code has moved on.

I'm not sure, but I do recall that some people in the Pande lab were eager to give constant-pH a try, so I mainly wanted to document the current state of the code. If we end up cutting a pre-release for them, it may as well feature this.